app/controllers/admin/getStudent.py

```diff
@@ -0,0 +1,27 @@
+from app.logic.searchStudents import searchVolunteers
+from app.controllers.admin import admin_bp
+from flask import flash,redirect, url_for, request, abort
+from app.controllers.main import main_bp
+from peewee import *
+
+
+@admin_bp.route('/searchStudents/<query>', methods = ['GET'])
+def searchStudents(query):
+    '''Accepts user input and queries the database returning results that matches user search'''
+    query = query.strip()
+    search = query.upper()
+    splitSearch = search.split()
+    try:
+        return searchVolunteers(query)
+
+    except Exception as e:
+        return e, 500

```

Comment:
If you're just going to return the error and a 500 code, you don't need to catch the exception. That's the default behavior for exceptions. You catch it if you want to do something like display a friendlier message, or log the output to the console rather than show the user (or do some other logic with it)